pipe/li7200/li7200_data_source_trino.yaml

```diff
@@ -45,7 +45,7 @@ transform:
       export GEN_OUTPUT_DIR=$interimDir/$SOURCE_TYPE/$year/$month/$day
       export REQUESTS_CA_BUNDLE=/etc/pki/tls/cert.pem
       mkdir -p $GEN_OUTPUT_DIR
-      /usr/src/app/genscript/genparquet.py --storesitename --codec gzip
+      /usr/src/app/genscript/genparquet.py --storesitename --codec gzip --truncperiod second
```

Comment:
Are you sure you want to truncate to the second? Isn't li-7200 code at 20 Hz?

Reply:
Good catch, will revert

Reply:
seems need your approval again with a new check-in. Thanks!